doc/nixcon.md

```diff
@@ -43,3 +43,74 @@ Organisers are using:
 If the organisers cannot reach consensus on important decisions,
 they are escalated to the Steering Committee.
 Decisions relating to sponsorships are always escalated to the Steering Committee.
+
+## Sponsorship tiers
+
+This should be displayed on the website.
+
+### Non-sponsorship donation (0+ EUR)
+- Supporting the conference and official Nix projects
+- Confirmation of donation via OpenCollective
+
+### Bronze (1000+ EUR):
+- Linked logo on the website
+- Linked shout-out or repost from official social media accounts
+- Possibility to distribute your own stickers/swag/merch at the venue
+- Tickets: 1 corporate tickets[^1]
+
+[^1]: Only corporate tickets come with company name/logo recognition on the badge. Corporate tickets can also be bought individually when ticket sales open.
+
+### Silver (3000+ EUR)
```

Comment:
This one seems overpriced / under-provisioned compared to the Brronze and Gold. Maybe 2k?

Reply:
Would be great to also get input from the @NixOS/marketing-team on the tiers. For comparison, here's previous years:

- NixCon 2023 tiers: https://discourse.nixos.org/t/nixcon-2023-sponsorship-tracks/29550
  - Effective sponsors: https://2023.nixcon.org/#sponsors
- NixCon 2022 tiers: https://discourse.nixos.org/t/nixcon-2022-sponsorship-opportunities/20603
  - Effective sponsors: https://2022.nixcon.org/#sponsors
- NixCon 2020 tiers: https://2020.nixcon.org/sponsoring.html
  - Effective sponsors: https://2020.nixcon.org/#sponsors

Reply:
There is little logistical complexity for organizers for the extras in Silver. The big jump is providing a 5min slot and a booth in Gold. I'd think we should encourage the Silver over Bronze with a lower price point because the unit-of-effort/extra-price is fairly low.

Not a huge deal and not a blocker, just a small optimization.